rotten_tomatoes/correlation_analysis.py

```diff
@@ -0,0 +1,26 @@
+import seaborn as sns
+import pandas as pd
+import numpy as np
+
+class CorrelationAnalysis:
+    
+    def __init__(self, d):
+        self.data = d
+        
+    def corr_matrix(self, subset = None):
```

Comment:
Would add an assert that the columns of subset are in data.

Reply:
If the columns aren't a subset you hit a pandas assert failure which felt more transparent to the user. Do you think an additional assert is necessary?

Reply:
Ok sure, that's fair.